VisDataProvider.cpp

```diff
@@ -179,6 +179,17 @@ int VisDataProvider::pull() {
     return 0;
 }
 
+bool VisDataProvider::waitConnection(int s) const {
+    ALOGV("waitConnectionMs");
```

Comment:
Maybe it's better to log same name as function name, or log message with "%d ms".